api/app/jobs/rdps_sfms.py

```diff
@@ -146,13 +147,14 @@ async def run(self):
         logger.info("Begin download and storage of RDPS gribs.")
 
         # grab the start time.
-        start_time = datetime.now()
+        start_time = datetime.now(timezone.utc)
         with get_write_session_scope() as session:
             rdps_grib = RDPSGrib(session)
             await rdps_grib.process()
             await rdps_grib.apply_retention_policy(DAYS_TO_RETAIN)
+            await compute_and_store_precip_rasters(start_time)
```

Comment:
I'd suggest adding a check of `rdps_grib.files_downloaded > 0` to prevent re-computing and re-storing rasters.

Reply:
Oh wait, I don't think my question is valid because we aren't always computing and storing rasters based on the currently downloaded RDPS grib files, we start off using yesterday's downloaded files. We still might want some sort of check to make sure we're not doing unnecessary work, but I don't think this is the place for it.

Reply:
We might also run into issues with PR deployments if this cronjob run and yesterday's RDPS gribs haven't been downloaded yet.